### PR TITLE
scripts/get_latest_release.py: fixed bug where JSON was being improperly accessed

### PR DESCRIPTION
**Pull request by dseevr**
Signed-off-by: Bill Robinson <dseevr@users.noreply.github.com>

### CODE REVIEW

File: scripts/get_latest_release.py [hunk 1]
```diff
@@ -1,22 +1,32 @@
...
    r = requests.get("https://api.github.com/repos/contiv/install/releases")
    if r.status_code != HTTP_OK:
       # Return a known version
-      print "1.0.0-beta.3"
+      print DEFAULT_VERSION
+      sys.exit(1)
+
+   try:
+      releases = json.loads(r.content)
+      print releases[0]['name']
```
Comment (dseevr) [inline scripts/get_latest_release.py]: `/releases` has an array

File: scripts/get_latest_release.py [hunk 2]
```diff
@@ -1,22 +1,32 @@
...
+   except:
+      print DEFAULT_VERSION
       sys.exit(1)
 try:
    releases = json.loads(r.content)
-   print releases[0]['name']
+   print releases['name']
```
Comment (dseevr) [inline scripts/get_latest_release.py]: `/releases/latest` does not

File: scripts/get_latest_release.py [hunk 3]
```diff
@@ -1,22 +1,32 @@
...
+
 # Try to get the latest release version
 r = requests.get("https://api.github.com/repos/contiv/install/releases/latest")
 if r.status_code != HTTP_OK:
    # Get all the releases (include pre-releases) and pick the first of them
    r = requests.get("https://api.github.com/repos/contiv/install/releases")
    if r.status_code != HTTP_OK:
       # Return a known version
-      print "1.0.0-beta.3"
+      print DEFAULT_VERSION
```
Comment (gaurav-dalvi) [inline scripts/get_latest_release.py]: Print some message before that defaulting to 1.0.0 version because of failure.

Comment (dseevr) [inline scripts/get_latest_release.py]: I don't want to change the behavior of the existing script, though I do want to have a conversation about the "defaulting to some version" behavior.  Waiting for @neelimamukiri to get back from PTO to ask why it's currently that way

**Pull request merged**